### ISSUE TITLE
Add a flag to 'tkn pipeline start` to return PipelineRun name

### ISSUE THREAD

**Issue by jland-redhat** [tektoncd/cli#1426]
### Feature request

A flag in which the `tkn pipeline start` return's just the name of the PipelineRun

### Use case

Currently trying to make a basic smoke test pipeline that will run through and call an existing pipeline multiple times in sequence with different parameters. Currently when you run the command you get a message that is nice for a user where it gives you a command for following the PipelineRun's log. 

But it would be nice if we could include a flag that would modify the output to just return the name of the PipelineRun so I could have a subsequent command for following the logs without having to pull the name using a regex expression to extract it.

**Comment by liangyuanpeng** [comment]
That makes sense, how about param of `--return-name` ?

**Comment by nikhil-thomas** [comment]
i would love to have this feature. 🧑‍💻 
i think what we need is a `procelain` flag like we have in `git`.
```
--porcelain

Give the output in an easy-to-parse format for scripts.
```

/remove-lifecycle rotten

**Comment by ppitonak** [comment]
`tkn pipeline start` already has a param `--output` that can accept values `yaml` or `json`. IMHO we should do the same that kubectl/oc is doing, i.e. the `--output` param should accept value `name` which would print something like
```
$ tkn pipeline start my-pipeline ...
pipelinerun/my-pipeline-run-mqg9t
```

**Comment by nikhil-thomas** [comment]
oh. great, then let us close this issue. 🧑‍💻 🕺 
/close

**Comment by nikhil-thomas** [comment]
@piyush-garg 👆

**Comment by ppitonak** [comment]
@nikhil-thomas https://github.com/tektoncd/cli/pull/1433 hasn't been merged yet

**Comment by digglife** [comment]
This issues has been solved by #1540, so it can be closed.
